scripts/basic-workflow-tests.sh

```diff
@@ -110,8 +110,14 @@ test_smoke() {
 
 test_init() {
     task Initializing ilab
-    ilab config init --non-interactive
 
+    if [ "$FULLTRAIN" -eq 1 ]; then
+        # TODO Only cuda for now
+        step Setting train-profile for GPU accelerated training
+        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-a10.yaml"
+    else
+        [ -f config.yaml ] || ilab config init --non-interactive
```

Comment:
I don't think `[ -f config.yaml ]` is ever going to work given that we explicitly use `"${CONFIG_HOME}/instructlab/config.yaml"` for the config file. @RobotSail to keep me honest but I raised it in another and it was acked. It's minimal but why re-introduce it?

Reply:
good point, I noticed there was a rebase so I kept what I had. Will put back the regular `ilab config init --non-interactive`

Reply:
+1 @leseb , we need to use "${CONFIG_HOME}/instructlab/config.yaml" here